cli/studio.py

```diff
@@ -65,6 +65,13 @@ def _build_single_scenario(clean, allow_offset_map, scenario):
 
     scenario_root = Path(scenario)
     map_net = str(scenario_root / "map.net.xml")
+    if not os.path.isfile(map_net):
+        click.echo(
+            "FILENOTFOUND: no reference to map.net.xml was found in {}.  "
+            "Please make sure the path passed is a valid SUMO scenario with SUMO network file (map.net.xml) required "
+            "for scenario building.".format(str(scenario_root))
+        )
+        return
```

Comment:
The above seems ok in the very short term ("for now") but we should start thinking about generalizing this to support scenarios with other map formats (including scenarios where the map is from a server instead of a file) very soon.

Reply:
I have generalized this to support new map formats like `.xodr` for `OpenDRIVE `in the `opendrive-road-network` branch.

Reply:
I think this might drive the need for a cleaner way of switch on different types of scenarios maps. It seems reasonable if it comes in through the `OpenDRIVE` branch.